docs/release-source/release/fake-timers.md

```diff
@@ -43,34 +43,95 @@ set of features (Sinon uses it under the hood) and was previously extracted from
 
 
 #### `var clock = sinon.useFakeTimers(now);`
 
-As above, but rather than starting the clock with a timestamp of 0, start at the provided timestamp.
+As above, but rather than starting the clock with a timestamp of 0, start at the provided timestamp `now`.
 
 *Since `sinon@2.0.0`*
 
 You can also pass in a Date object, and its `getTime()` will be used for the starting timestamp.
 
-#### `var clock = sinon.useFakeTimers([now, ]prop1, prop2, ...);`
+#### `var clock = sinon.useFakeTimers(config);`
 
-Sets the clock start timestamp and names functions to fake. If the first argument is not numeric, it sets the clock to 0 and treats all arguments as names of functions to fake.
+Installs fake timers with the following options:
 
-Possible functions are `setTimeout`, `clearTimeout`, `setInterval`, `clearInterval`, `setImmediate`, `clearImmediate` and `Date`.  Any functions not listed continue to use the original version of the function, including the actual time for the timestamp. This can have surprising results and should be done with care.
+Parameter | Type | Description
+--------- | ---- | ------------
+`config.now` | Number/Date | installs lolex with the specified unix epoch (default: 0)
+`config.toFake` | String[] | an array with explicit function names to fake. *By default lolex will automatically fake all methods **except** `process.nextTick`*. You could, however, still fake `nextTick` by providing it explicitly
+`config.shouldAdvanceTime` | Boolean | tells lolex to increment mocked time automatically based on the real system time shift (default: false)
 
-Note that if no functions are listed, the default behavior is to replace all eligible functions.
+****Important note:*** when faking `nextTick`, normal calls to `process.nextTick()` would not execute automatically as they would during normal event-loop phases. You would have to call either `clock.next()`, `clock.tick()`, `clock.runAll()` or `clock.runToLast()`. Please refer to the [lolex](https://github.com/sinonjs/lolex) documentation for more information.*
 
+#### Examples:
 
-#### `clock.tick(ms);`
+Installs fake timers at January 1st 2017 and fakes `setTimeout` and `process.nextTick` only:
+```js
+var clock = sinon.useFakeTimers({
+                now: 1483228800000, 
+                toFake: ["setTimeout", "nextTick"] 
+            }); 
 
-Tick the clock ahead `ms` milliseconds.
+var called = false;
 
-Causes all timers scheduled within the affected time range to be called.
+process.nextTick(function () {
+    called = true;
+});
 
+clock.runAll(); //forces nextTick calls to flush synchronously
+assert(called); //true
+```
+
+Install at the same date, advancing the fake time automatically (default is every 20ms), causing timers to be fired automatically without the need to `tick()` the clock:
+
+```js
+var clock = sinon.useFakeTimers({
+                now: 1483228800000, 
+                shouldAdvanceTime: true 
+            });
+
+setImmediate(() => {
```

Comment:
Minor: since we're going for **wide** adoption, targeting ES5.1, I would prefer it if the examples were also ES5.1.